pq-crypto/kyber_r3/kyber_r3_kem.c

```diff
@@ -0,0 +1,127 @@
+#include <stddef.h>
+#include <stdint.h>
+#include "params.h"
+#include "symmetric.h"
+#include "verify.h"
+#include "indcpa.h"
+#include "tls/s2n_kem.h"
+#include "utils/s2n_safety.h"
+#include "pq-crypto/s2n_pq_random.h"
+#include "pq-crypto/s2n_pq.h"
+
+/*************************************************
+* Name:        crypto_kem_keypair
+*
+* Description: Generates public and private key
+*              for CCA-secure Kyber key encapsulation mechanism
+*
+* Arguments:   - unsigned char *pk: pointer to output public key
+*                (an already allocated array of CRYPTO_PUBLICKEYBYTES bytes)
+*              - unsigned char *sk: pointer to output private key
+*                (an already allocated array of CRYPTO_SECRETKEYBYTES bytes)
```

Comment:
You're right - all of the `CRYPTO_*` references in the comments should be renamed to the appropriate `KYBER_512_R3_*` constant. They're just leftover from the import. (I think) NIST gave the authors of the submission code guidelines for naming so that the submissions would be consistent, but we've renamed things when importing the code into s2n so that it fits our codebase better.